src/Pages/Home/Home.jsx

```diff
@@ -1,48 +1,15 @@
-import React, { useEffect, useState } from 'react';
-import GetHalls from '../../Api/PublicApi/public_api';
-import HomeSlider from './Carousel/HomeSlider';
+import React from 'react';
 import Features from './Features/Features';
-import styles from './home.module.css';
+import './home.css';
 
 function Home() {
-  const [Halls, setHalls] = useState(null);
-  const [Error, setError] = useState(null);
-
-  useEffect(() => {
-    GetHalls(setError, setHalls);
-  }, []);
-
```

Comment:
Good job enhancing the homepage it looks beautiful but it is static, it would be a good idea to make it more responsive.